Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1050,6 +1052,21 @@ inline ScrollSnapStop BuilderConverter::convertScrollSnapStop(BuilderState&, con
     return fromCSSValue<ScrollSnapStop>(value);
 }
 
+inline std::optional<class ScrollbarColor> BuilderConverter::convertScrollbarColor(BuilderState& builderState, const CSSValue& value)
+{
+    if (is<CSSPrimitiveValue>(value)) {
+        ASSERT(value.valueID() == CSSValueAuto);
+        return std::nullopt;
+    }
+
+    ASSERT(value.isPair());
+    const auto& pair = downcast<CSSValuePair>(value);
+
+    std::optional<class ScrollbarColor> result = *(new class ScrollbarColor(builderState.colorFromPrimitiveValueWithResolvedCurrentColor(downcast<CSSPrimitiveValue>(pair.first())), builderState.colorFromPrimitiveValueWithResolvedCurrentColor(downcast<CSSPrimitiveValue>(pair.second()))));
```

Comment:
Why is it using `new` and they call copy actor ? It should just be a local stack value.